server/rpc_server.go

```diff
@@ -404,11 +404,35 @@ func (s *rpcServer) Start() error {
 	s.opts.Address = ts.Addr()
 	s.Unlock()
 
-	go ts.Accept(s.accept)
+	exit := make(chan bool, 1)
```

Comment:
Could this be a `chan struct{}`, given it’s a signal to exit?

Reply:
Absolutely could be, this is just directly lifted from upstream as a sort of manual cherrypick.

Reply:
For reference https://github.com/asim/go-micro/blob/v0.13.0/server/rpc_server.go#L402

Reply:
Ah, right. Fair enough 👍 

LGTM too!